docs/sources/reference/components/otelcol/otelcol.exporter.syslog.md

```diff
@@ -0,0 +1,346 @@
+---
+canonical: https://grafana.com/docs/alloy/latest/reference/components/otelcol/otelcol.exporter.syslog/
+aliases:
+  - ../otelcol.exporter.syslog/ # /docs/alloy/latest/reference/components/otelcol.exporter.syslog/
+description: Learn about otelcol.exporter.syslog
+title: otelcol.exporter.syslog
+---
+
+# otelcol.exporter.syslog
+
+{{< docs/shared lookup="stability/public_preview.md" source="alloy" version="<ALLOY_VERSION>" >}}
+
+`otelcol.exporter.syslog` accepts logs from other `otelcol` components and writes them over the network using the syslog protocol. 
+It supports syslog protocols [RFC5424][RFC5424] and [RFC3164][RFC3164] and can send data over `TCP` or `UDP`.
+
+{{< admonition type="note" >}}
+`otelcol.exporter.syslog` is a wrapper over the upstream OpenTelemetry Collector `syslog` exporter.
+Bug reports or feature requests will be redirected to the upstream repository, if necessary.
+{{< /admonition >}}
+
+You can specify multiple `otelcol.exporter.syslog` components by giving them different labels.
+
+[RFC5424]: https://www.rfc-editor.org/rfc/rfc5424
+[RFC3164]: https://www.rfc-editor.org/rfc/rfc3164
+
+## Usage
+
+```alloy
+otelcol.exporter.syslog "LABEL" {
+  endpoint = "HOST"
+}
+```
+
+<!-- 
+The next few paragraphs were copied from the OTel docs:
```

Comment:
Unless there's a really good reason to copy in external documentation, I'd strongly prefer that we just link to the external docs.  We could paraphrase things... if that helps?

If we do need include external docs that are copied verbatim, then we need to give attribution and make sure we stick within the license of the doc.

Reply:
I had paraphrased, but then I saw a copy-paste block in the `otelcol.processor.probabilistic_sampler` doc and assumed it was a standard, looks like it's the opposite 🙃 . I do think having the exact copied tables of attribute mapping is important, but I'll work on a refactor in the morning that does less copying and more attribution.

Reply:
I updated the docs to consolidate the details & link out, much more of a re-write now so I didn't add full attribution.